src/torcontrol.cpp

```diff
@@ -292,10 +297,52 @@ static std::map<std::string,std::string> ParseTorReplyMapping(const std::string
             if (ptr == s.size()) // unexpected end of line
                 return std::map<std::string,std::string>();
             ++ptr; // skip closing '"'
-            /* TODO: unescape value - according to the spec this depends on the
-             * context, some strings use C-LogPrintf style escape codes, some
-             * don't. So may be better handled at the call site.
+            /**
+             * Escape value. Per https://spec.torproject.org/control-spec section 2.1.1:
```

Comment:
Note that a quote is escaped iff it is preceded by an odd number of backslashes.